apicapi/tests/unit/common/test_apic_common.py

```diff
@@ -310,9 +311,8 @@ def set_up_mocks(self):
         }
         self.vlan_ranges = ['physnet0', 'physnet1:100:199']
         self.mocked_parser = mock.patch.object(
-            cfg, 'MultiConfigParser').start()
-        self.mocked_parser.return_value.read.return_value = [apic_mock_cfg]
-        self.mocked_parser.return_value.parsed = [apic_mock_cfg]
+            apic_cfg, '_parse_files').start()
+        self.mocked_parser.return_value = [apic_mock_cfg]
```

Comment:
should probably add a UT to ensure that multiple config files can actually be read/parsed correctly